devcycle_python_sdk/cloud_client.py

```diff
@@ -61,15 +73,15 @@ def variable(self, user: DevCycleUser, key: str, default_value: Any) -> Variable
         try:
             variable = self.bucketing_api.variable(key, user)
         except CloudClientUnauthorizedError as e:
-            logger.warning("DevCycle: SDK key is invalid, unable to make cloud request")
+            logger.warning("DevCycle SDK key is invalid, unable to make cloud request")
             raise e
         except NotFoundError:
-            logger.warning("DevCycle: variable not found: %s", key)
```

Comment:
Do we have a prefix handler setup for this to show devcycle as the source of the log?

Reply:
Well all of the loggers do have names that should include the path so developers should be able to filter and control the log output based on the logger names